services/net/command/CommandProcess.cs

```diff
@@ -15,6 +15,11 @@ public class CommandProcess : ICommandProcess
     /// get/set - When the process was created.
     /// </summary>
     public DateTime CreatedOn { get; set; }
+
+    /// <summary>
+    /// get - Data included in the command process.
+    /// </summary>
+    public Dictionary<string, object> Data { get; } = new Dictionary<string, object>();
```

Comment:
Allows storing data with the process (i.e. filename) so that when the stop command is called it knows which file it started with.